core/templates/dev/head/pages/footer_js_libs.html

```diff
@@ -1,5 +1,2 @@
-{% if not DEV_MODE %}
-  <script src="/third_party/generated/js/third_party.min.js"></script>
-{% else %}
-  <script src="/third_party/generated/js/third_party.js"></script>
-{% endif %}
+<script src="/third_party/generated/js/third_party.js" ng-if="inDevMode"></script>
```

Comment:
Can you please point to the scope where `inDevMode` is defined? (filename with line no. will work!)

Reply:
/core/templates/dev/head/pages/admin/Admin.js line 32

Reply:
`/core/templates/dev/head/pages/admin/Admin.js` is not used in most pages `/core/templates/dev/head/pages/Base.js` is used mostly. But I'm not sure if the variable is available in footer_js_libs.js

Reply:
Hmmm, I am not sure if that's the right thing to do. I defer this question to @vojtechjelinek and @DubeySandeep.

Reply:
@nithusha21 I found this method doesn't work and it still tries to load unwanted script, we want the power of jinja (server rendering) to keep this working, is there anything else we can try ?